Examples/Io/Performance/ActsExamples/Io/Performance/CKFPerformanceWriter.cpp

```diff
@@ -185,10 +218,14 @@ ActsExamples::ProcessCode ActsExamples::CKFPerformanceWriter::writeT(
         inputFeatures[2] = trajState.chi2Sum * 1.0 / trajState.NDF;
         // predict if current trajectory is 'duplicate'
         bool isDuplicated = m_cfg.duplicatedPredictor(inputFeatures);
+	// Add to number of duplicated particles
+	if (isDuplicated) m_nTotalDuplicateTracks ++;
```

Comment:
Here you are basing your computation of the duplicate rate on an neural network that predict if a track is a duplicate or not. Is that really what you want to do ?